tasks/main.yml

```diff
@@ -12,10 +12,29 @@
   apt: pkg=buildkite-agent state=present
 
 - name: Configure Buildkite
-  template: src=buildkite-agent.cfg.j2
-            dest="{{ buildkite_agent_conf_dir }}/buildkite-agent.cfg"
-            owner=buildkite-agent
-            group=buildkite-agent
-            mode=0600
+  template:
+    src: "buildkite-agent.cfg.v{{ buildkite_agent_major_version }}.j2"
+    dest: "{{ buildkite_agent_conf_dir }}/buildkite-agent.cfg"
+    owner: buildkite-agent
+    group: buildkite-agent
+    mode: "0600"
   notify:
     - Restart Buildkite
+
+- name: Turn off the installed service
+  service:
+    name: buildkite-agent
+    enabled: no
+    state: stopped
+
+- name: Create systemd template unit
+  copy:
+    src: /lib/systemd/system/buildkite-agent.service
+    dest: /lib/systemd/system/buildkite-agent@.service
```

Comment:
This file seems to be included in the BK agent package. Also, `copy` without `remote_src: True` leads to the copying of `src` from the local file system to the remote.

I think that we can remove this entire task.

Reply:
Huh, I hadn't noticed that, and just blithely followed BK's docs for doing this. Ok; removed.